net/README.md

```diff
@@ -20,27 +30,156 @@ $ aws configure
 ```
 More information on AWS CLI configuration can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-quick-configuration)
 
-### Metrics configuration (Optional)
-Ensure that `$(whoami)` is the name of an InfluxDB user account with enough
-access to create a new InfluxDB database.  Ask mvines@ for help if needed.
+## Metrics configuration (Optional)
+Metrics collection relies on 2 environment variables that are patched to the remote nodes:
+ * `RUST_LOG` to enable metrics reporting in principle
+ * `SOLANA_METRICS_CONFIG` to tell agave where to log the metrics
+
+### Preparation
+> [!NOTE]
+> Anza employees should follow the guide in notion to set up the influxDB account.
+
+ * Ensure that `${host}` is the host name of the InfluxDB you can access, for example `https://metrics.solana.com:9999`
+ * Ensure that `${user}` is the name of an InfluxDB user account with enough
+rights to create a new InfluxDB database, for example `solana`.
+
+### To set up the metrics
+* Go to ./net/ in agave repo
+* Run `./init-metrics.sh -c testnet-dev-${user} ${user} `
```

Comment:
do we actually need to run this before running `./gce.sh create`?

Reply:
need -no, but if not gce will complain about metrics being broken. Also in my experience it makes sense to do this before running gce as it can fail if DB perms are not set up correctly.

Reply:
Might be worth clarifying that people should only run this once and not each time they want to use gce clusters.

Reply:
done